services/powchain/src/main/java/tech/pegasys/artemis/services/powchain/PowchainService.java

```diff
@@ -62,11 +62,13 @@
   private String provider;
 
   private String depositSimFile;
+  private TimeProvider timeProvider;
 
   public PowchainService() {}
 
   @Override
   public void init(ServiceConfig config) {
+    timeProvider = config.getTimeProvider();
```

Comment:
(nit) might want to switch this line to:
```
this.timeProvider = config.getTimeProvider();
```
To be in line with other member variables.

Reply:
```suggestion
    this.timeProvider = config.getTimeProvider();
```